### PR TITLE
Corrected wording and added punctuation

### PR DESCRIPTION
**Pull request by cakesoft-khushi**
Corrected wording and added punctuation where required.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -7,21 +7,21 @@ Solidity
     :align: center
 
 Solidity is an object-oriented, high-level language for implementing smart
-contracts. Smart contracts are programs which govern the behaviour of accounts
+contracts. Smart contracts are programs that govern the behaviour of accounts
 within the Ethereum state.
 
-Solidity was influenced by C++, Python and JavaScript and is designed to target
+Solidity was influenced by C++, Python, and JavaScript and is designed to target
```
Comment (chriseth) [inline docs/index.rst:13]: We usually don't use the Oxford comma.

Comment (cakesoft-khushi) [reply docs/index.rst:13]: It has been used in some cases. Its use isn't consistent so added it.

Comment (chriseth) [reply docs/index.rst:13]: Sure! I have the impression, though, that there should be more places where we do not sue it than there are where we us eit.

**Comment by chriseth** [comment]
Please rebase on top of develop instead of using merge commits.

**Pull request closed without merging**